torch/csrc/jit/passes/quantization/insert_quant_dequant.cpp

```diff
@@ -890,12 +890,35 @@ void propagateQuantizationOps(Block* block) {
         }
       }
     } else {
+      // For this category of ops, we need to
```

Comment:
What is the mechanism for warning/providing errors for quantization? In this case, it will be good to error out with a specific message.
```
if (n->outputs().size() > 1) {
        // Factoring out dequantize for if blocks with multiple outputs	        // Factoring out dequantize for if blocks with multiple outputs
        // is not supported right now	        // is not supported right now
        continue;	        continue;
      }	      }
```

Reply:
this should probably be a warning since we might not need this functionality when there are multiple outputs, and the result would be incorrect if we do need the support.

Reply:
In general when it's a clear case we don't support, we'll throw an error